src/models/matrix-config.ts

```diff
@@ -0,0 +1,44 @@
+export type MatrixDefaultConfig = {
+  xURL: string;
+  yURL: string;
+  zURL: string;
+  xysURL: string;
+  xFacetColumn: string;
+  yFacetColumn: string;
+  zFacetColumn: string;
+  xSource: AxisSource;
```

Comment:
Let's use `any` for the `xSource`, `ySource`, and `zSource`. Its value could be a string or array of elements. The typing should be something that we're eventually going to add to ermrestjs.